src/status_im/utils/money.cljs

```diff
@@ -30,26 +30,16 @@
 (defn bignumber [n]
   (when n
     (try
-      (.toBigNumber dependencies/web3-prototype (normalize (str n)))
+      (new dependencies/BigNumber (normalize (str n)))
```

Comment:
i remember we have a problem with BigNumber here? is it solved?

Reply:
we overwrote the web3 BigNumber version, that is just using BigNumber directly